### ISSUE TITLE
More compact version

### ISSUE THREAD

**Issue by ff6347** [vogelino/periodic-table-data-visualization#2]
Maybe not for know but think of a more compact version where we don't have to scroll

**Comment by vogelino** [comment]
Yeah that's true. I did not figured out how to display the 118 elements on a single non-scrollable vis. I'll think about it. If you got an idea let me know! :+1:

**Comment by ff6347** [comment]
No. No solution here right now. I'll think about it.
